projects/collection-editor-library/src/lib/components/assign-page-number/assign-page-number.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { EditorService } from '../../services/editor/editor.service';
+import { TreeService } from '../../services/tree/tree.service';
+import * as _ from 'lodash-es';
+
+@Component({
+  selector: 'lib-assign-page-number',
+  templateUrl: './assign-page-number.component.html',
+  styleUrls: ['./assign-page-number.component.scss']
+})
+export class AssignPageNumberComponent implements OnInit {
+
+  toolbarConfig: any = {};
+  pageId = 'pagination';
+  treeData: any;
+  questions: any;
```

Comment:
variable, not used? why do we need this?

Reply:
we have declared for the functionalities to be used right now we have not implemented the functionalities